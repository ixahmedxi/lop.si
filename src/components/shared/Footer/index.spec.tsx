import { render } from 'test-utils'
import { Footer } from '.'

describe('Container component', () => {
  it('matches snapshot', () => {
    const { container } = render(<Footer />)
    expect(container).toMatchSnapshot()
  })
})
