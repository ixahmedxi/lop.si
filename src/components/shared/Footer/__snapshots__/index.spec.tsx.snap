// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Container component matches snapshot 1`] = `
.emotion-3 {
  box-sizing: border-box;
  margin: 0;
  min-width: 0;
  font-size: 10px;
  margin: 0;
  padding: 0;
  line-height: 2;
  text-align: center;
  padding-bottom: 16px;
}

@media screen and (min-width:576px) {
  .emotion-3 {
    font-size: 10px;
  }
}

@media screen and (min-width:768px) {
  .emotion-3 {
    font-size: 12px;
  }
}

.emotion-0 {
  box-sizing: border-box;
  margin: 0;
  min-width: 0;
  color: var(--theme-ui-colors-primary,#8093f2);
  -webkit-text-decoration: underline;
  text-decoration: underline;
  cursor: pointer;
  font-weight: bold;
}

<div>
  <p
    class="emotion-3"
  >
    Made with 
    <svg
      fill="none"
      height="1em"
      stroke="currentColor"
      stroke-linecap="round"
      stroke-linejoin="round"
      stroke-width="2"
      viewBox="0 0 24 24"
      width="1em"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="M20.84 4.61a5.5 5.5 0 0 0-7.78 0L12 5.67l-1.06-1.06a5.5 5.5 0 0 0-7.78 7.78l1.06 1.06L12 21.23l7.78-7.78 1.06-1.06a5.5 5.5 0 0 0 0-7.78z"
      />
    </svg>
     by
     
    <a
      aria-label="my twitter profile"
      class="emotion-0"
      href="https://twitter.com/ixahmedxii"
      rel="noopener noreferrer"
      target="_blank"
    >
      Ahmed Elsakaan
    </a>
     
    <br />
    By using our services you accept our
     
    <a
      aria-label="terms and conditions page"
      class="emotion-0"
    >
      Terms & Conditions
    </a>
     
    and
     
    <a
      aria-label="privacy policy page"
      class="emotion-0"
    >
      Privacy Policy
    </a>
  </p>
</div>
`;
